src/code/z_camera.c

```diff
@@ -8,12 +8,24 @@ s32 Camera_ChangeModeFlags(Camera* camera, s16 mode, u8 flags);
 s32 Camera_QRegInit(void);
 s32 Camera_UpdateWater(Camera* camera);
 
-#define RELOAD_PARAMS \
-    (camera->animState == 0 || camera->animState == 0xA || camera->animState == 0x14 || R_RELOAD_CAM_PARAMS)
+#define CAM_GET_RO_DATA(type, camera) &((type*)camera->paramData)->roData // Read-Only Data
+#define CAM_GET_RW_DATA(type, paramData) &((type*)paramData)->rwData      // Read-Write Data
```

Comment:
```suggestion
#define CAM_GET_RW_DATA(type, camera) &((type*)camera->paramData)->rwData // Read-Write Data
```
😄 

but I'm starting to wonder if having two macros is all that useful and if
`CAM_(GET_)DATA(Normal1, camera)->roData`/rwData
would be better?
It would generalize the `ONEPOINT_CS_INFO` macro

Reply:
Could you elaborate on this actually? I know there are fancier things you can do with macros in C that I still don't understand/haven't learned. What would that macro look like? Just to brainstorm

Reply:
Ah I see. Perhaps, it's an interesting idea. To note, I'm not satisfied with the name `paramData`, and am looking for something more meaningful. That new name could then also translate to make a more interesting name than `CAM_DATA` too